doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -166,7 +166,7 @@ This section provides detailed lists of changes by :ref:`application <applicatio
 Asset Tracker v2
 ----------------
 
-|no_changes_yet_note|
+  * Removed nrf7002ek DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because disabling uart1 is done through the shield configuration.
```

Comment:
```suggestion
  * Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because disabling UART1 is done through the shield configuration.
```